locales/de_DE/pcsx2_Iconized.po

```diff
@@ -644,6 +698,11 @@ msgid ""
 "CPU requirements substantially. Lower values will reduce the CPU load "
 "allowing lightweight games to run full speed on weaker CPUs."
 msgstr ""
+"Modifiziert den emulierten Emotion-Enigne CPU-Takt. Höhere Werte können die "
```

Comment:
Emotion-Engine

Reply:
Oops, I'll look at it, thanks for pointing out.

Reply:
Fixed